solr/solr-ref-guide/src/learning-to-rank.adoc

```diff
@@ -418,6 +500,14 @@ Learning-To-Rank is a contrib module and therefore its plugins must be configure
 </transformer>
 ----
 
+* Declaration of the `[interleaving]` transformer.
++
+[source,xml]
+----
+<transformer name="interleaving" class="org.apache.solr.ltr.response.transform.LTRInterleavingTransformerFactory">
+</transformer>
```

Comment:
minor/subjective: could shorten since there's no parameters

```
<transformer name="interleaving" class="org.apache.solr.ltr.response.transform.LTRInterleavingTransformerFactory"/>
```